recipe/meta.yaml

```diff
@@ -30,13 +26,12 @@ requirements:
     - setuptools
     - cython
     - cysignals
-    - libsemigroups >=0.2.2
+    - libsemigroups >=0.3.0
     - nose
-    # pytest-cython # not yet on conda-forge
 
   run:
     - python
-    - libsemigroups >=0.2.2
+    - libsemigroups >=0.3.0
```

Comment:
Looks like `cysignals` is a runtime requirement as well. You have to add it here